wire-ios/Wire-iOS.xcodeproj/project.pbxproj

```diff
@@ -9057,6 +9075,24 @@
 			shellPath = /bin/sh;
 			shellScript = "rm -rf \"${SOURCE_ROOT}\"/SnapshotResults/*\n";
 		};
+		E6C01B382BF7788A0059EC62 /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "# Wire-iOS\n../scripts/run-sourcery.sh --config ./Tests/Sourcery/config.yml\n\n# WireAPI\n../scripts/run-sourcery.sh --config ../WireAPI/Sourcery/config.yml\n";
```

Comment:
Of course `WireDataModel` etc. are missing currently, as it should be only one example.